### ISSUE TITLE
Cannot get values with TS.RANGE, with and without streaming.

### ISSUE THREAD

**Issue by adrianfime27** [RedisGrafana/grafana-redis-datasource#254]
### Describe the bug

I'm testing grafana to make a streaming graphs, but only can with **TS.GET**, but also this command get the last value. Works fine to monitoring industrial process, but the key with me, is make graph with an history, be able to review the values on the past dates. For that reason, I want use **TS.RANGE** to see the values.
Or if not, with what function can I do it?
### Version

Redis v6.0.9
Grafana v8.2.5 
The module Redis Time Series is installed on the server.
![ts](https://user-images.githubusercontent.com/11671423/150009631-886a7deb-d417-424f-8a93-544100208f5d.png)
### Screenshots

So, If search by TS.GET this is the graph.
![uht_ph1](https://user-images.githubusercontent.com/11671423/150010286-2925e1e3-816e-4199-8713-46cd5381c7f5.png)

And, If search by TS.RANGE to see the history, this is the another graph.
![uht_ph2](https://user-images.githubusercontent.com/11671423/150010385-a3be9755-b09f-4566-943e-134bc39feb48.png)

**Comment by mikhailredis** [comment]
Hello @adrianfime27,

According to the documentation for TS.RANGE command (https://redisgrafana.github.io/redis-datasource/redis-timeseries/TS-RANGE/) Streaming should be supported. Have you tried to use TS.RANGE without Streaming? For monitoring historical data I would advice to refresh dashboard instead of using Streaming, which is for real-time.

**Comment by adrianfime27** [comment]
Yes, also trying without streaming, as I mentioned in the title of the bug.

**Comment by mikhailredis** [comment]
@adrianfime27 I am sorry, I missed that. Do you see any data returned when running `TS.RANGE` using Redis-cli?

**Comment by adrianfime27** [comment]
Yes! With the _redis-cli_ all works fine. But when try in grafana, what the picture shows happens.

**Comment by adrianfime27** [comment]
@mikhailredis Hi?

**Comment by mikhailredis** [comment]
Hello @adrianfime27, I am sorry for the delay.

Could you please try the latest version to see if it makes the difference: https://grafana.com/grafana/plugins/redis-datasource/.
I tested the latest version with TS.RANGE and it's working as expected on my test dataset with similar versions.

If the latest version won't show anything, could you please run MONITOR and check the command from the Grafana, is it the same as you execute from `redis-cli`?

**Comment by adrianfime27** [comment]
Hello @mikhailredis, I'm sorry for the delay. But is with the last version of plugin or grafana?

**Comment by mikhailredis** [comment]
@adrianfime27 The latest version of the plugin, which was recently updated in the Grafana Marketplace.

**Comment by mikhailredis** [comment]
@adrianfime27 Redis Data Source is used to return Redis Data.

Redis Application and Redis Explorer application plugins add additional functionality for managing Data Sources.

**Comment by adrianfime27** [comment]
The problem continues, I have updated the Redis Data Source plugin, but it still does not work.
![git1](https://user-images.githubusercontent.com/11671423/153291234-60c0819c-e363-49b0-8008-c99f0019463d.png)
I'm running Grafana from a docker-compose, I'll also show you how I do it if you see something strange.
`version: "3.4"
services:
  redis:
    container_name: redistimeseries
    image: redislabs/redistimeseries:edge
    ports:
      - 6379:6379
    command:
      - redis-server
      - /data/redis.conf
    volumes:
      - /home/adrian/DockStation/redists-grafana/conf:/data/

  grafana:
    container_name: grafana
    image: grafana/grafana:latest
    ports:
      - "3000:3000"
    environment:
      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
      - GF_AUTH_ANONYMOUS_ENABLED=true
      - GF_AUTH_BASIC_ENABLED=false
      - GF_ENABLE_GZIP=true
      - GF_INSTALL_PLUGINS=redis-datasource, redis-app
      - GF_SERVER_ROOT_URL=http://192.168.2.47:3000
      - GF_SECURITY_ADMIN_PASSWORD=xxxxxxxx`

**Comment by mikhailredis** [comment]
@adrianfime27 Everything looks correct.

Adding Redis team @K-Jo and @danni-m to assist further.

**Comment by kramik1** [comment]
I have this same problem. I get the keys from doing a TS.mrange but no data. I am sad to see this is in production and promoted but this issue has been open for so long. I am using the latest grafana and data source plugin. The cli command does the same in the grafana tool but works on the cli itself. Is it having problems parsing the output? It appears to be related to timeseries module queries since the redis dashboard works itself. The grafana forum has no history of this issue. I gather redis timeseries isn't a popular data source yet.

**Comment by mikhail-vl** [comment]
@kramik1 Redis team was supposed to take over the Redis data source and application plugins in January, but they are not interested in supporting them.

**Comment by slorello89** [comment]
Interesting - oddly if you have streaming enabled it seems to to continuously ask for the same time over and over again - and rather than appending the results from the timeseries onto the end of the data frame that it retrieved already, marks the whole thing as a new field - if you set the output to data frame it does display the graph as you'd expect - but that graph will be static as it's just getting the same data over and over again. Will dig into it.

**Comment by slorello89** [comment]
Hi @adrianrjh - I've looked into this, and I've concluded that TS.RANGE/TS.MRANGE are not appropriate for a streaming use-case. A streaming use-case expects each update to bear a single discrete data point that will be updated added to the end of the timeseries tracked by grafana. Ranges, by definition pull back all the data points over the time range.

What you want to use here is an auto-refresh on the dashboard, that will update with a new range when the period elapses. see the upper right hand corner of your dashboard for the auto-refresh feature.

![image](https://github.com/RedisGrafana/grafana-redis-datasource/assets/42971704/2289cced-9c96-419e-909e-c9ad2f910303)

There are instances within Redis Timeseries where streaming is appropriate, e.g. `TS.GET`, and `TS.MGET`, in that case, you would simply need to provide a value label to  the query:

![image](https://github.com/RedisGrafana/grafana-redis-datasource/assets/42971704/ce9b0de4-a0c3-4770-aa91-3a68603f3871)

hopefully this is helpful!
